package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "roosterjs",
-    "version": "8.26.1",
+    "version": "8.26.2",

```

Comment:
Change the package.json triggers a new version of roosterJS and we only do this in every 2 weeks. 
We usually open a PR just to bump the version.

Reply:
I am following the debugging guide and it said to do that, what should I do then?

Reply:
As Julia said, we normally do this every two weeks, unless this is an urgent fix and want to be integrated immediately. You don't need to touch this file in this change.